inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseProfileDeserializationTest.java

```diff
@@ -49,13 +53,22 @@ public void checkThatDeserializationWorks() throws JsonParseException, JsonMappi
         
         KnowledgeBaseMapping referenceMapping = new KnowledgeBaseMapping(classIri, subclassIri, typeIri, descriptionIri, label , propertyTypeIri);
         KnowledgeBaseProfile referenceProfile = new KnowledgeBaseProfile();
+
+        KnowledgeBaseAccess referenceAccess = new KnowledgeBaseAccess();
+        referenceAccess.setAccessUrl(url);
+        referenceAccess.setAccessType(accessType);
+
         referenceProfile.setMapping(referenceMapping);
         referenceProfile.setName(name);
-        referenceProfile.setSparqlUrl(url);
+        referenceProfile.setAccess(referenceAccess);
+        referenceProfile.setType(type);
         
         String test_yaml = "test_profile:\n    "
```

Comment:
I'd suggest to please the test profile into a file under `src/test/resources` which is then loaded by the test.